Packs/Phishing/Playbooks/Phishing_Investigation_-_Generic_v2_README.md

```diff
@@ -52,4 +52,4 @@ There are no outputs for this playbook.
 
 ## Playbook Image
 ---
-![Phishing_Investigation_Generic_v2](https://raw.githubusercontent.com/demisto/content/master/docs/images/playbooks/Phishing_Investigation_Generic_v2.png)
+![Phishing_Investigation_Generic_v2](https://github.com/demisto/content/raw/master/Packs/Phishing/doc_files/Phishing_Investigation_Generic_v2.png)
```

Comment:
I think it is better to use the commit hash. That way it is not affected by cases that the file moves.
```suggestion
![Phishing_Investigation_Generic_v2](https://raw.githubusercontent.com/demisto/content/4000f5d617a0929a78095a6bb3aa90279b9ee527/Packs/Phishing/doc_files/Phishing_Investigation_Generic_v2.png)
```